src/amo/components/RatingsByStar/index.js

```diff
@@ -29,13 +28,37 @@ type Props = {|
 
 type InternalProps = {|
   ...Props,
+  clientApp: string,
   dispatch: DispatchFunc,
   errorHandler: ErrorHandlerType,
   groupedRatings?: GroupedRatingsType,
   i18n: I18nType,
   location: ReactRouterLocationType,
+  siteLang: string,
 |};
 
+type WrappingLinkProps = {|
+  styleClass: string,
+  children: React.Node,
+  rowProps: {
+    title: string,
+    link: string,
+  },
+|};
+
+function WrappingLink(props: WrappingLinkProps) {
+  const { styleClass, children, rowProps } = props;
+
+  if (Object.keys(rowProps).length > 0) {
+    return (
+      <Link to={rowProps.link} className={styleClass} title={rowProps.title}>
```

Comment:
Is there a reason you need to use `Link` from `react-router-dom` rather than from `amo/components/Link`?